### ISSUE TITLE
Top-level README should list included projects by subdir.

### ISSUE THREAD

**Issue by mrutkows** [apache/incubator-openwhisk-devtools#27]
As this repo. now houses 2 independent "deployments" (i.e., Compose and Kube) and now we have a PR for node-test subdir. we need the top-level README to let people know all the sub-projects that are structured under this "devtools" repo.

**Comment by mrutkows** [comment]
The PR I referenced above is here: https://github.com/apache/incubator-openwhisk-devtools/pull/26/files

**Comment by DanLavine** [comment]
I just submitted another PR for the Kube work which makes OpenWhisk deployable on a Kube version 1.5. Once that is merged in, I will submit another PR to have the main README give some details about the Kube work.

Also, I have a know of a couple enhancements that need to be made. Will open another issue to track all of the enhancements after I close the first one [here](https://github.com/apache/incubator-openwhisk-devtools/issues/14).
